cypress/utils/apps/query_enhancements/inspect.js

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { QueryLanguages } from './constants';
+
+const visualizationTitlesWithNoInspectOptions = [
+  '[Flights]Controls',
+  '[Flights]MarkdownInstructions',
+  '[Flights]Delays&Cancellations',
+];
+
+const visualizationTitlesWithInspectOptions = [
+  '[Flights]AirlineCarrier',
+  '[Flights]FlightCountandAverageTicketPrice',
+  '[Flights]TotalFlights',
+  '[Flights]AverageTicketPrice',
+];
+
+/**
+ * Gets Date and returns string of date in format Jan 24, 2025 @ 16:20:08.000
```

Comment:
should add example for input and output

Reply:
should not Jan 24, 2025 @ 16:20:08.000 in overall description. we can use it as example as line 24. check my comment for formatValue and add more details.